### ISSUE TITLE
WISH: extend adapter to allow creation of events

### ISSUE THREAD

**Issue by and0nna** [iobroker-community-adapters/ioBroker.ical#334]
This is a wish, not a bug/issue :-)

Would it be possible to extend the functionality of the adapter to allow for creation of events.
This would enable logging of certain activities in the calendar and reporting timewise afterwards (e.g. xyz was active on Monday between 03:00 and 16:15).

Tried to make use of nodered for this functionality, but this would require a running nodered environment.

Many thanks in advance for your support!

**Comment by Apollon77** [comment]
Hm ... this is already possible? YOu can device these events in the config and then the calendar entries are parsed and events created. Or what exactly you mean?

**Comment by and0nna** [comment]
I read in the doc, the adapter allows reading of events (URL...ics, local ics file) and parses them.
There is an EVENTS 'tab' in the admin/config screen, Are you refering to it?
Selecting a datapoint where 'events' are stored would transfer the event to the calc? 
I am guessing, as I didn't find a description of the tab, nor further explanation of event creation in the doc. (Or I overlooked it)

Basically, I would like to create events in e.g. google calendar based on data in ioBroker, for example 'Door has been opened at 7.23am and closed at 7.33'. based on that time span I would like to create an cal entry spanning this period incl. some further details like which key opend the door.; car has been parked at home starting 10am till 02pm, ....,  something like that.

**Comment by Apollon77** [comment]
Oookkkk .... This adapter just _ready_ ical files ... there is no write support ... so with this you idea will not work with this adapter ... or do I get something wrong?
Or you need "ical write support"?

**Comment by and0nna** [comment]
mhhh ... OK. Maybe ical write isn't possible with this adapter.
Did not deep dive into the sources and the APIs used, as my programming and javascript capabilities are limited and the adapter framework is totally new to me.
seems i need to rethink my approach, or start understanding the adapter framework + javascript :-)
